### PR TITLE
Update Beaumont config file

### PR DESCRIPTION
**Pull request by SamGuay**
@arnaudbore, here's the newest config file. 

Feel free to ask questions regarding my naming convention, I am open to improving it in any way possible :)

### CODE REVIEW

File: beaumont/config_HOF_2021.json [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  "dupMethod": "dup",
+  "descriptions": [
+    {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "criteria": {
+        "SeriesDescription": "anat_T1w"
+      }
+    },
+    {
```
Comment (arnaudbore) [inline beaumont/config_HOF_2021.json]: You can add a second T1w with customLabels "rec-norm" to extract:
```
"ImageType": ["ORIGINAL", "PRIMARY", "M", "ND", "NORM"]
```
then you need to add to extract the original T1w:
```
   "ImageType": ["ORIGINAL","PRIMARY","M","NORM","DIS3D","DIS2D"],
```

File: beaumont/config_HOF_2021.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      "sidecarChanges": {
+        "MTState": true
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-1_flip-1_part-mag_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0075,
```
Comment (arnaudbore) [inline beaumont/config_HOF_2021.json]: I would use EchoNumber instead of EchoTime since it can slightly vary.

**Review by arnaudbore — APPROVED** [review]
Ok rename the new one just config_HOF.json and it should be good LGTM :)

File: beaumont/config_HOF_2021.json [hunk 3]
```diff
@@ -0,0 +1,188 @@
+{
+  "dupMethod": "dup",
+  "descriptions": [
+    {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "criteria": {
+        "SeriesDescription": "anat_T1w"
+      }
+    },
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
    {
      "dataType": "anat",
      "modalityLabel": "T1w",
      "criteria": {
        "SeriesDescription": "anat_T1w"
        "ImageType": ["ORIGINAL","PRIMARY","M","NORM","DIS3D","DIS2D"],
      }
    },
     {
      "dataType": "anat",
      "modalityLabel": "T1w",
      "customLabels": "rec-norm",
      "criteria": {
        "SeriesDescription": "anat_T1w_ND"
        "ImageType": ["ORIGINAL", "PRIMARY", "M", "ND", "NORM"]
      }
    },
```

Comment (SamGuay) [reply beaumont/config_HOF_2021.json]: Like this @arnaudbore ?

File: beaumont/config_HOF_2021.json (hunk 2 above)
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 1,
```

File: beaumont/config_HOF_2021.json [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-1_flip-1_part-phase_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0075,
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 1,
```

File: beaumont/config_HOF_2021.json [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-2_flip-1_part-mag_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0175,
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 2,
```

File: beaumont/config_HOF_2021.json [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+          "PHASE"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-2_flip-1_part-phase_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0175,
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 2,
```

File: beaumont/config_HOF_2021.json [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-4_flip-2_part-mag_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.01875,
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 2,
```

File: beaumont/config_HOF_2021.json [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+          "PHASE"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-4_flip-2_part-phase_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.01875,
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 2,
```

File: beaumont/config_HOF_2021.json [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-3_flip-2_part-phase_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.00875,
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 1,
```

File: beaumont/config_HOF_2021.json [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+          "PHASE"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-3_flip-2_part-mag_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.00875,
```
Comment (SamGuay) [inline beaumont/config_HOF_2021.json]:
```suggestion
        "EchoNumber": 1,
```

**Review by SamGuay — COMMENTED** [review]
Here's the changes as suggested @arnaudbore.

File: beaumont/config_HOF_2021.json (hunk 2 above)
Comment (SamGuay) [reply beaumont/config_HOF_2021.json]: I change with echo numbers, but note that echo 1 and 2 are different between the 2 sequences. That's why I went with `echo-3` and `echo-4` to make sure we _remember_ they differ.

File: beaumont/config_HOF.json [hunk 11]
```diff
@@ -1,98 +1,188 @@
...
-    ]
+  "dupMethod": "dup",
+  "descriptions": [
+    {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "criteria": {
+        "SeriesDescription": "anat_T1w"
+      }
+    },
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
    {
      "dataType": "anat",
      "modalityLabel": "T1w",
      "criteria": {
        "SeriesDescription": "anat_T1w"
        "ImageType": ["ORIGINAL","PRIMARY","M","NORM","DIS3D","DIS2D"],
      }
    },
     {
      "dataType": "anat",
      "modalityLabel": "T1w",
      "customLabels": "rec-norm",
      "criteria": {
        "SeriesDescription": "anat_T1w_ND"
        "ImageType": ["ORIGINAL", "PRIMARY", "M", "ND", "NORM"]
      }
    },
```

File: beaumont/config_HOF.json [hunk 12]
```diff
@@ -1,98 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-2_flip-1_part-mag_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0175,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 2,
```

File: beaumont/config_HOF.json [hunk 13]
```diff
@@ -1,98 +1,188 @@
...
+      "sidecarChanges": {
+        "MTState": true
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-1_flip-1_part-mag_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0075,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 1,
```

File: beaumont/config_HOF.json [hunk 14]
```diff
@@ -1,98 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-1_flip-1_part-phase_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0075,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 1,
```

File: beaumont/config_HOF.json [hunk 15]
```diff
@@ -1,98 +1,188 @@
...
+          "PHASE"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-2_flip-1_part-phase_acq-PDw",
+      "criteria": {
+        "EchoTime": 0.0175,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 2,
```

File: beaumont/config_HOF.json [hunk 16]
```diff
@@ -1,98 +1,188 @@
...
+          "PHASE"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-3_flip-2_part-mag_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.00875,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 1,
```

File: beaumont/config_HOF.json [hunk 17]
```diff
@@ -1,98 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-4_flip-2_part-mag_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.01875,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 2,
```

File: beaumont/config_HOF.json [hunk 18]
```diff
@@ -1,98 +1,188 @@
...
+          "NORM"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-3_flip-2_part-phase_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.00875,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 1,
```

File: beaumont/config_HOF.json [hunk 19]
```diff
@@ -1,98 +1,188 @@
...
+          "PHASE"
+        ]
+      }
+    },
+    {
+      "dataType": "swi",
+      "modalityLabel": "MEGRE",
+      "customLabels": "echo-4_flip-2_part-phase_acq-T1w",
+      "criteria": {
+        "EchoTime": 0.01875,
```
Comment (SamGuay) [inline beaumont/config_HOF.json]:
```suggestion
        "EchoNumber": 2,
```

**Review by SamGuay — COMMENTED** [review]
Same changes, with the file renamed :upside_down_face:

File: beaumont/config_HOF.json [hunk 20]
```diff
@@ -1,98 +1,198 @@
...
-            }
-        }
-    ]
+  "dupMethod": "dup",
+  "descriptions": [
+    {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "criteria": {
+        "SeriesDescription": "anat_T1w"
```
Comment (arnaudbore) [inline beaumont/config_HOF.json]:
```suggestion
        "SeriesDescription": "anat_T1w",
```

File: beaumont/config_HOF.json [hunk 21]
```diff
@@ -1,98 +1,198 @@
...
+        "ImageType": ["ORIGINAL","PRIMARY","M","NORM","DIS3D","DIS2D"],
+      }
+    },
+     {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "customLabels": "rec-norm",
+      "criteria": {
+        "SeriesDescription": "anat_T1w_ND"
```
Comment (arnaudbore) [inline beaumont/config_HOF.json]:
```suggestion
        "SeriesDescription": "anat_T1w_ND",
```

**Review by arnaudbore — COMMENTED** [review]
final comments

File: beaumont/config_HOF.json [hunk 22]
```diff
@@ -1,98 +1,198 @@
...
-        }
-    ]
+  "dupMethod": "dup",
+  "descriptions": [
+    {
+      "dataType": "anat",
+      "modalityLabel": "T1w",
+      "criteria": {
+        "SeriesDescription": "anat_T1w",
+        "ImageType": ["ORIGINAL","PRIMARY","M","NORM","DIS3D","DIS2D"],
```
Comment (arnaudbore) [inline beaumont/config_HOF.json]:
```suggestion
        "ImageType": ["ORIGINAL","PRIMARY","M","NORM","DIS3D","DIS2D"]
```

**Pull request merged**